backends/candle/src/models/flash_modernbert.rs

```diff
@@ -1,14 +1,15 @@
 use std::collections::HashMap;
 
 use crate::flash_attn::flash_attn_varlen;
-use crate::layers::{apply_rotary, get_cos_sin, get_inv_freqs, LayerNorm, Linear};
+use crate::layers::{get_cos_sin, get_inv_freqs, LayerNormNoBias, Linear};
```

Comment:
This was already landed in the orginal PR, it's a good way to handle the fact that the layer norm never has any bias here.